pgsql-flexible-server.tf

```diff
@@ -72,3 +84,14 @@ resource "azurerm_postgresql_flexible_server_configuration" "pgsql_server_config
   server_id = azurerm_postgresql_flexible_server.pgsql_server.id
   value     = each.value.value
 }
+
+
+resource "azurerm_postgresql_flexible_server_active_directory_administrator" "pgsql_adadmin" {
+  count               = var.set_ad_admin != null ? 1 : 0
```

Comment:
enable by default please assuming it works all the time

Reply:
Sure? enabling by default will require all the dependant infra to pass in ad_pricipal_name and type.  So didn't want to break the pipeline

Reply:
I've added some suggestions so those aren't needed